python/hsfs/feature_group.py

```diff
@@ -425,7 +448,9 @@ def select_features(
         # Returns
             `Query`. A query object with all features of the feature group.
         """
-        query = self.select_except(self.primary_key + [self.event_time])
+        query = self.select_except(
```

Comment:
Function documentation needs to be updated to mention that the function does not select `foreign_keys` also.

!!! info
        This method does not select the primary key, foreign key and event time of the feature group.
        Use `select_all` to include them.
        Note that primary keys do not need to be included in the query to allow joining
        on them.